main.js

```diff
@@ -228,6 +228,17 @@ function createJitsiMeetWindow() {
 
     mainWindow.webContents.setWindowOpenHandler(windowOpenHandler);
 
+    // Filter out x-frame-options and CSP to allow loading jitsi via the iframe API
+    // Resolves https://github.com/jitsi/jitsi-meet-electron/issues/285
+    mainWindow.webContents.session.webRequest.onHeadersReceived((details, callback) => {
+        delete details.responseHeaders['x-frame-options'];
+        delete details.responseHeaders['content-security-policy'];
```

Comment:
I thought we had established that some CSPs do work?

Reply:
Good point, I changed the approach to only filter out the frame-ancestors CSP and keep everything else in place. With that, CSP would eg. still protect against loading 3rd party resources in the iframe itself if this is enforced via CSP.